### DISCUSSION TITLE
Problem about units used in Moltres

### DISCUSSION THREAD

**Discussion by js-jixu** [arfc/moltres#203] [Q&A Getting started] [answered]
Hi, everyone. I have a question about the unit used in Moltres.

Usually the energy unit is J, then the energy unit used by the group constant in Moltres is J or eV ? My understanding is that the unit of mass in Moltres is kg and the unit of length is cm, is this understanding correct?

Thanks a lot.

**Comment by smpark7** [comment, marked as answer]
The units for energy and length are J and cm. These specifications are based on the assumption that the cross sections are provided in cm^-1 and KAPPA (energy per fission) is provided in MeV.

All other units depend on whichever units you provide your other material properties (e.g. whether density is in g/cm3 or kg/cm3, whether temperature is in Celsius or Kelvin.)

**Reply by js-jixu** [reply]
OK, thank you very much!
